test/test_tp_compliance.c

```diff
@@ -95,13 +95,31 @@ void TestISOTPSendRecvMaxLen(void **state) {
     assert_memory_equal(UDSTpGetRecvBuf(srv, NULL), MSG, sizeof(MSG));
 }
 
+void TestISOTPFlowControlFrameTimeout(void **state) {
+    if (!IsISOTP()) {
+        skip();
+    }
+
+    // killing server so that no response is sent to client
+    ENV_TpFree(srv);
```

Comment:
I added this because for the test is it necessary not to have a server responding to the request so we can get the desired timeout on the ISOTP socket. Because of the timeout error the value of `ret` is `<0`, which was not the case before setting the flag on the socket.